agent/api/task/task_windows_test.go

```diff
@@ -109,7 +109,6 @@ func TestPostUnmarshalWindowsCanonicalPaths(t *testing.T) {
 				},
 			},
 		},
-		StartSequenceNumber: 42,
```

Comment:
This field has been removed as part of https://github.com/aws/amazon-ecs-agent/pull/3723